blockbuster/src/programs/bubblegum/mod.rs

```diff
@@ -201,7 +199,12 @@ impl ProgramParser for BubblegumParser {
                     InstructionName::UnverifyCollection => {
                         b_inst.payload = Some(build_collection_verification_payload(keys, false)?);
                     }
-                    InstructionName::Unknown => {}
+                    InstructionName::UpdateMetadata => {
+                        let args = UpdateMetadataInstructionArgs::try_from_slice(&outer_ix_data)?;
+                        let update_args = args.update_args;
+                        b_inst.payload = Some(Payload::UpdateMetadata { update_args });
+                    }
+                    InstructionName::Unknown | InstructionName::SetDecompressibleState => {}
```

Comment:
Do we need line `207` given that we have `_ => {}` in the line below that does the same thing?

Reply:
removed